content/en/docs/setup/independent/control-plane-flags.md

```diff
@@ -21,6 +21,9 @@ The `extraArgs` field consist of `key: value` pairs. To override a flag for a co
 
 1.  Add the appropriate fields to your configuration.
 2.  Add the flags to override to the field.
+3.  Update the configuration by `kubeadm upgrade apply --config config.yml` 
+
+You can also add `extraVolumes` to mount additional volumes if necessary (e.g. specifying a configuration file for cloud provider gce).
```

Comment:
```suggestion
You can also add `extraVolumes` to mount additional volumes if necessary (e.g. including configuration files that are relevant to your cloud provider).
```

Reply:
there are already examples here:
https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2#hdr-Kubeadm_init_configuration_types

the godoc ref pages are linked in this document already.

Reply:
> there are already examples here:
> https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2#hdr-Kubeadm_init_configuration_types
> 
> the godoc ref pages are linked in this document already.

This link is good, but not in this document.

The link in this document is https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#ClusterConfiguration

which is less informative

Reply:
i think it's fine to change the link to https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2